infrastructure/src/main/java/org/corfudb/infrastructure/log/statetransfer/batch/Batch.java

```diff
@@ -0,0 +1,34 @@
+package org.corfudb.infrastructure.log.statetransfer.batch;
+
+import com.google.common.collect.ImmutableList;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * A batch of addresses, small enough to get transferred within one rpc call and
+ * an optional destination server where the corresponding data is located.
+ */
+@AllArgsConstructor
+@Getter
+@ToString
+@EqualsAndHashCode
+@Builder
+public class Batch {
```

Comment:
"Batch" is a too generic name, we should change it.

Reply:
Was fixed